modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,130 @@
+* This update adds v1 API support to all CLI commands associated with task, pipeline, pipeline run, and task run resources. Tekton CLI works fine with both v1beta1 and v1 APIs for these resources.
+* This update adds support for an object type parameter in the `star` and `describe` commands.
+
+
+[id="operator-new-features-1-10_{context}"]
+=== Operator
+
+* This update adds a `default-forbidden-env` parameter in optional pipeline properties. The parameter includes forbidden environment variables that should not be propagated if provided through pod templates.
+* This update adds support for custom logos in Tekton Hub UI. To add a custom logo, set the value of the `customLogo` parameter to base64 encoded URI of logo in the Tekton Hub CR. 
+* This update increments the version number of the git-clone task to 0.9.
+* This update adds the `resource-verification-mode` and `enable-provenance-in-status` parameters in the `feature-flag` config map for pipelines.
+* With this update, you can define both the `keep` and `keep-since` parameters simultaneously for a TaskRun or PipelineRun resource in the pruner configuration file.
+
+
+[id="chains-new-features-1-10_{context}"]
+=== Tekton Chains
+
+* This update adds annotations and labels to the `PipelineRun` and `TaskRun` attestations.
+* This update adds a new format `slsa/v1`, which generates the same provenance as that generated when requesting in the `in-toto` format.
+* With this update, Sigstore features are moved out from the experimental features.
+* With this update, the `predicate.materials` function includes image URI and digest information from all steps and sidecars for a `TaskRun`.
+
+
+[id="tekton-hub-new-features-1-10_{context}"]
+=== {tekton-hub}
+
+* This update supports install, upgrade, or downgrade of Tekton resources with API versions `tekton.dev/v1beta1` and `tekton.dev/v1` on the cluster.
+* This update supports adding a custom logo in place of the {tekton-hub} logo in UI.
+* This update extends the `tkn hub install` command functionality by adding a flag `--type artifact`, which fetches resources from the Artifact Hub and install them on your cluster.
+* This update adds support tier, catalog, and org information as labels to the resources being installed  from Artifact Hub to your cluster.
+
+
+[id="pac-new-features-1-10_{context}"]
+=== {pac}
+
+* This update enhances incoming webhook support. For a GitHub application installed on the cluster, you do not need to provide the `git_provider` specification for an incoming webhook. Instead, {pac} detects the secret and use it for the incoming webhook.
+* This update enhances incoming webhook support. You do not need to provide a personal token when using the GitHub App driver. You only need to have the repository CR into the namespace that matches with the GitHub URL and an incoming webhook secret. {pac} automatically generates temporary tokens for the GitHub API operations.
```

Comment:
I feel both are pointing to same feature

Reply:
I also felt both are pointing to the same feature, but then the descriptions looked a bit different. This is why kept both of them. Now, I am removing the second bullet.

Reply:
Looks good to me